tools/istio-iptables/pkg/dependencies/implementation_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dependencies
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestIsXTablesLockError(t *testing.T) {
+	xtableError := []struct {
+		name    string
+		errMsg  string
+		errCode XTablesExittype
+		want    bool
+	}{
+		{
+			name:    "no error",
+			errMsg:  "Another app is currently holding the xtables lock.",
```

Comment:
The difference between this and the last test is error code.
However the error message is the same, that does not look right.

Reply:
Yeah this won't happen in reality. Updated.